### PR TITLE
docs: add a warning before running create-external-cluster-resources.sh

### PR DESCRIPTION
**Pull request by MaXinjian**
Signed-off-by: Ma Xinjian <maxj.fnst@cn.fujitsu.com>

**Description of your changes:**
If we don't export RGW_POOL_PREFIX firstly, RGW_POOL_PREFIX will be set to default value, which is "default". Add a warning in document to give user instructions.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [x] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

**Review by leseb — APPROVED** [review]

**Review by leseb — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please update your commit message to include the description of the change.

**Comment by MaXinjian** [comment]
> Thanks for your contribution. Please update your commit message to include the description of the change.

Done

**Review by leseb — APPROVED** [review]

**Comment by leseb** [comment]
@subhamkrai please review the doc formating.

File: Documentation/ceph-cluster-crd.md [hunk 1]
```diff
@@ -1266,6 +1266,27 @@ export ROOK_EXTERNAL_ADMIN_SECRET=AQC6Ylxdja+NDBAAB7qy9MEAr4VLLq4dCIvxtg==
...
+
+```console
+$ ceph osd pool ls
+my-store.rgw.control
+my-store.rgw.meta
+my-store.rgw.log
+my-store.rgw.buckets.index
+my-store.rgw.buckets.non-ec
+my-store.rgw.buckets.data
+```
```
Comment (subhamkrai) [inline Documentation/ceph-cluster-crd.md:1282]:
```suggestion
**Example**:

```console
ceph osd pool ls
```
>```
>my-store.rgw.control
>my-store.rgw.meta
>my-store.rgw.log
>my-store.rgw.buckets.index
>my-store.rgw.buckets.non-ec
>my-store.rgw.buckets.data
>```
```

**Review by subhamkrai — CHANGES_REQUESTED** [review]

File: Documentation/ceph-cluster-crd.md [hunk 2]
```diff
@@ -1266,6 +1266,27 @@ export ROOK_EXTERNAL_ADMIN_SECRET=AQC6Ylxdja+NDBAAB7qy9MEAr4VLLq4dCIvxtg==
 
 If the Ceph admin key is not provided, the following script needs to be executed on a machine that can connect to the Ceph cluster using the Ceph admin key.
 On that machine, run `cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh`.
+
+> **WARNING**: Before executing the script, RGW_POOL_PREFIX should be exported so as to match the RGW that has been built in Rook. You can get existing RGW by running `ceph osd pool ls`.
```
Comment (BlaineEXE) [inline Documentation/ceph-cluster-crd.md:1270]: I think some more clarity would be good. Which Rook are you referring to in "the RGW that has been built in Rook"? 

What should users do if there are no pools in the external cluster?

File: Documentation/ceph-cluster-crd.md (hunk 1 above)
Comment (MaXinjian) [reply Documentation/ceph-cluster-crd.md]: Done

File: Documentation/ceph-cluster-crd.md (hunk 2 above)
Comment (MaXinjian) [reply Documentation/ceph-cluster-crd.md]: I have updated, please review again.

**Review by subhamkrai — APPROVED** [review]

**Pull request merged**